drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -159,9 +159,11 @@ def active
   end
 
   def households_including_wip
-    household_ids = enrollments_including_wip.pluck(:household_id)
-
-    Hmis::Hud::Household.where(HouseholdID: household_ids, data_source_id: data_source_id)
+    # correlated subquery for performance
+    cp_t = Hmis::Hud::ClientProject.arel_table
+    hh_t = Hmis::Hud::Household.arel_table
+    Hmis::Hud::Household.where(data_source_id: data_source_id).
+      where(client_projects.where(cp_t[:HouseholdID].eq(hh_t[:HouseholdID])).arel.exists)
```

Comment:
Does this also need the data_source_id on the join?  I think it won't be an issue as long as there's only one HMIS data source, but HouseholdID isn't unique across data sources.

Reply:
Since we are within a project here, we use  its data source to filter the households on 165
```ruby
Hmis::Hud::Household.where(data_source_id: data_source_id)
```
Sorry if I'm missing something obvious but is there a way that wouldn't filter correctly?